Dockerfile.riff.build

```diff
@@ -0,0 +1,4 @@
+FROM heroku/pack:18-build
+
+USER root
```

Comment:
Do we need another `USER heroku` beneath the next `RUN`? Does this image start the same as `18-build`? The last `USER` directive is important sometimes, so I just want to make sure we aren't going to accidentally be running `exec` commands as `root`.

`docker run -it heroku/pack:18-riff-build whoami`